bees/emailbee/emailbee.go

```diff
@@ -23,7 +23,6 @@ package emailbee
 
 import (
 	"net"
-
```

Comment:
While a valid change, try to keep unrelated/separate changes in a different pull request.